Composer/packages/extensions/visual-designer/src/components/nodes/steps/ChoiceInput.tsx

```diff
@@ -21,7 +21,7 @@ export const ChoiceInputChoices = ({ choices }) => {
   }
 
   return (
-    <div data-testid="ChoiceInput" css={{ padding: '0 0 8px 45px' }}>
+    <div data-testid="ChoiceInput" css={{ padding: '0 0 16px 29px' }}>
```

Comment:
can this value be calculated from some constants?

Reply:
@yeze322 I'm not sure how to answer that... the goal is that ideally things are all multiples of 4. 

I'm not a math guy, but happy to try and help if there is some sort of ratio you want to make that seems like a good idea... or if switching to SVG files makes life easier for you?

Reply:
@DesignPolice Thanks Marc! I think your design has already covered those pixel numbers. Programmatically, we need some code optimization to align to your design.